pkgs/development/ocaml-modules/routes/default.nix

```diff
@@ -2,14 +2,14 @@
 
 buildDunePackage rec {
   pname = "routes";
-  version = "1.0.0";
+  version = "2.0.0";
 
-  useDune2 = true;
+  duneVersion = "3";
   minimalOCamlVersion = "4.05";
```

Comment:
```suggestion
  minimalOCamlVersion = "4.08";
```

Reply:
> Drop support for OCaml 4.05-4.07

Reply:
So what? There are many packages in `nixpkgs` without upstream support.

Reply:
What I'm saying is that we should update `minimalOCamlVersion` to be correct

Reply:
I fear we disagree on what “correct” means.

Reply:
Can you please educate me in how this attribute is supposed to be used so that I can make better reviews in the future?
From the name I assume it's the oldest version that is supported by the library. But I guess this is wrong?

Reply:
> But I guess this is wrong?

This does not sound wrong.

Reply:
> > But I guess this is wrong?
> 
> This does not sound wrong.

If it's not wrong, why is it not correct to set the `minimalOCamlVersion` the the minimally supported version then?

Reply:
This is correct too.

Reply:
Then I don't understand what's wrong with my suggestion?

Reply:
What’s wrong with claiming that 4.05 is the “minimally supported version” of the `routes` library in `nixpkgs`?

Reply:
I guess this actually builds on older OCamls, so I guess it's fine. I'm going to reach out to @anuragsoni and see why it claims to have dropped support.